types/node/index.d.ts

```diff
@@ -1014,36 +1014,35 @@ interface IterableIterator<T> { }
 *                                               *
 *-----------------------------------------------*/
 declare module "buffer" {
-    export var INSPECT_MAX_BYTES: number;
-    var BuffType: typeof Buffer;
-    var SlowBuffType: typeof SlowBuffer;
+    export const INSPECT_MAX_BYTES: number;
+    const BuffType: typeof Buffer;
+    const SlowBuffType: typeof SlowBuffer;
     export { BuffType as Buffer, SlowBuffType as SlowBuffer };
 }
 
 declare module "querystring" {
-    export interface StringifyOptions {
+    interface StringifyOptions {
         encodeURIComponent?: Function;
     }
 
-    export interface ParseOptions {
+    interface ParseOptions {
         maxKeys?: number;
         decodeURIComponent?: Function;
     }
 
-    interface ParsedUrlQuery { [key: string]: string | string[] | undefined; }
+    interface ParsedUrlQuery { [key: string]: string | string[]; }
 
-    export function stringify<T>(obj: T, sep?: string, eq?: string, options?: StringifyOptions): string;
-    export function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): ParsedUrlQuery;
-    export function parse<T extends {}>(str: string, sep?: string, eq?: string, options?: ParseOptions): T;
-    export function escape(str: string): string;
-    export function unescape(str: string): string;
+    function stringify(obj?: {}, sep?: string, eq?: string, options?: StringifyOptions): string;
```

Comment:
I think the first argument is mandatory.

Reply:
Forgive the ghetto windows shell:

```
C:\Users\Simon>node -v
v10.10.0

C:\Users\Simon>node
> querystring.stringify(undefined)
''
> querystring.stringify()
''
>
```